EMPRESS/ass_6_calc_class.py

```diff
@@ -0,0 +1,43 @@
+
+a = 2
+b = 10
+
+# Create class
+class Calculator:
+    # adding self parameter is necessary, it is a reference
+    # to the current instance of the class.
+    def add(self,x,y): 
```

Comment:
You should add a comma between the arguments and the comma to conform to code style guidelines:

def add(self, x, y)